curriculum/challenges/english/25-front-end-development/lecture-working-with-the-dom-click-events-and-web-apis/673368b1cf26253212a3cfb2.md

```diff
@@ -10,6 +10,142 @@ dashedName: what-is-the-queryselectorall-method-and-when-should-you-use-it
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+What is the `querySelectorAll()` method, and when should you use it?
+
+Let's learn about the `querySelectorAll()` method.
+
+You can use this method to get a list of all the DOM elements that match a specific CSS selector.
+
+This is how you can call the `querySelectorAll()` method: call it on the document object and pass a string with the CSS selectors as an argument:
+
+```js
+document.querySelectorAll(selectors);
+```
+
+The argument must be a valid CSS selector string. Otherwise, a `SyntaxError` exception will be thrown.
```

Comment:
```suggestion
Call it on the `document` object and pass a string with the CSS selectors as an argument. The argument must be a valid CSS selector string. Otherwise, a `SyntaxError` exception will be thrown.
```